cpp/examples/mo_load_coefficients.cpp

```diff
@@ -105,7 +105,41 @@ void load_and_init(GasOpticsRRTMGP &kdist, std::string filename, GasConcs const
   } else {
     // Otherwise, it's a shortwave type
     realHost1d solar_src;
-    io.read( solar_src , "solar_source" );
+    if (io.varExists("solar_source")) {
+      io.read( solar_src , "solar_source" );
+    } else if (io.varExists("solar_source_quiet")) {
```

Comment:
This was copied from the eamxx/rrmpg copy of mo_load_coefficients. I did a diff between the files and this was the only meaningful difference.

Reply:
This needs to be revisited, but is beyond the scope of this pr.